Notes.md

```diff
@@ -6,4 +6,14 @@
 
 ```bash
 echo "This is interesting.!"
-```
\ No newline at end of file
+```
+
+Q) What is Git?
```

Comment:
Please change this `Q)` to a markdown element like `#` / `-` / `*` etc.

Reply:
The flow is that, you'll implement the suggested changes within this PR by adding more commits to this branch and then pushing them to remote.

After that's done, you hit `Resolve conversation` to indicate that this change request has been accomodated.